### PR TITLE
update on the wiki

### PR DESCRIPTION
**Pull request by Vanmc**
alright I'll get to into it really sorry for long reply got stuck on what gh-page branch means until I realize its about github branch XD silly me https://youtu.be/AcW533EHQXE here is for the playlist

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -610,6 +622,8 @@ Additional Falling Blocks:
 
 ```
 
+##
+
```
Comment (RoboMWM) [inline readme.md]: ?

**Comment by Vanmc** [comment]
Is there something wrong? You asked me to document the https://github.com/MLG-Fortress/ExtraHardMode/issues/34 Animal Crowd Control and I just did. I made a pull up since I can't directly edit the gh-page in your fork.

**Comment by RoboMWM** [comment]
You added a blank new section (where I made the line comment)?

**Comment by Vanmc** [comment]
Oh sorry I probably missed that part D:

**Pull request merged**